arcade/experimental/video_cv2.py

```diff
@@ -90,4 +111,30 @@ def on_update(self, delta_time: float):
                 self.texture.write(frame)
 
 
-CV2Player().run()
+class CV2PlayerView(arcade.View):
+    """
+    Can be used to add effects like rain to the background of the game.
+    Make sure to inherit this view and call super for `__init__`, `on_draw` and `on_update`.
+    """
```

Comment:
I think this docstring is unnecessary. It's really jist meant as an example.

Reply:
Updated https://github.com/pythonarcade/arcade/pull/1663/commits/a9f5133090d122f77571a90f14558a427a1d8d5a